src/java.desktop/share/classes/java/awt/Desktop.java

```diff
@@ -777,10 +711,6 @@ public void setOpenFileHandler(final OpenFilesHandler openFileHandler) {
      * {@code Info.plist}.
      *
      * @param printFileHandler handler
```

Comment:
@seanjmullan Can you please advice on some of these src file javadoc related clean-up review comments. Do they need to be handled in this PR? Some of them seem out-of-scope for jep486 PR.

Reply:
@honkar-jdk I'm inclined to leave it as because it's not the only method which doesn't have a blank line between `@param` and `@throw` in this file.

If it's worth taking care of, we may submit another bug to address it later.

Reply:
This does not need to be handled in this PR. In the majority of changes, we have really tried hard to avoid making unrelated changes, but sometimes a few snuck in, like moving package imports or perhaps fixing a typo here and there that was not specific to JEP 486. My opinion is that unless it is something that _really_ should be done as part of a more general technical debt or code cleanup exercise, then it is ok to let a few of these in and they don't have to be reverted.